tensorboard/components/tf_backend/requestManager.ts

```diff
@@ -187,9 +222,52 @@ export class RequestManager {
       req.onerror = function() {
         reject(new RequestNetworkError(req, url));
       };
-      req.send(formData);
+      if (requestOptions.body) {
+        req.send(requestOptions.body);
+      }
+      else {
+        req.send();
+      }
     });
   }
 }
+  
+function buildXMLHttpRequest(methodType: HttpMethodType, url: string,
+                             withCredentials? : boolean,
```

Comment:
Sorry for one last stylistic nit (we really should have a linter :( ): at least for consistency within this file, can you remove the whitespace before colon? `withCredentials?: boolean`